This is probably due to different output with zlib-ng
as the results look exactly the same.

-- q66

diff --git a/testsuite/gsk/nodeparser/empty-texture-scale.ref.node b/testsuite/gsk/nodeparser/empty-texture-scale.ref.node
index 6a77d0f..ec4d363 100644
--- a/testsuite/gsk/nodeparser/empty-texture-scale.ref.node
+++ b/testsuite/gsk/nodeparser/empty-texture-scale.ref.node
@@ -1,6 +1,6 @@
 texture-scale {
   bounds: 0 0 50 50;
-  texture: url("data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAoAAAAKCAYAAACNMs+9AAAAKUlEQVQYlWP8z3DmPwMaYGQwYUQX\
-Y0IXwAUGUCGGoxkYGBiweXAoeAYAz44F3e3U1xUAAAAASUVORK5CYII=\
+  texture: url("data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAoAAAAKCAYAAACNMs+9AAAAJklEQVQYlWP8z3DmPwMaYGQwYUQX\
+Y2IgEjANnEJGbILYPDgUPAMAz44F3Skt+skAAAAASUVORK5CYII=\
 ");
 }
diff --git a/testsuite/gsk/nodeparser/empty-texture.ref.node b/testsuite/gsk/nodeparser/empty-texture.ref.node
index b8027cb..18714b2 100644
--- a/testsuite/gsk/nodeparser/empty-texture.ref.node
+++ b/testsuite/gsk/nodeparser/empty-texture.ref.node
@@ -1,6 +1,6 @@
 texture {
   bounds: 0 0 50 50;
-  texture: url("data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAoAAAAKCAYAAACNMs+9AAAAKUlEQVQYlWP8z3DmPwMaYGQwYUQX\
-Y0IXwAUGUCGGoxkYGBiweXAoeAYAz44F3e3U1xUAAAAASUVORK5CYII=\
+  texture: url("data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAoAAAAKCAYAAACNMs+9AAAAJklEQVQYlWP8z3DmPwMaYGQwYUQX\
+Y2IgEjANnEJGbILYPDgUPAMAz44F3Skt+skAAAAASUVORK5CYII=\
 ");
 }
diff --git a/testsuite/gsk/nodeparser/texture-fail.ref.node b/testsuite/gsk/nodeparser/texture-fail.ref.node
index b8027cb..18714b2 100644
--- a/testsuite/gsk/nodeparser/texture-fail.ref.node
+++ b/testsuite/gsk/nodeparser/texture-fail.ref.node
@@ -1,6 +1,6 @@
 texture {
   bounds: 0 0 50 50;
-  texture: url("data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAoAAAAKCAYAAACNMs+9AAAAKUlEQVQYlWP8z3DmPwMaYGQwYUQX\
-Y0IXwAUGUCGGoxkYGBiweXAoeAYAz44F3e3U1xUAAAAASUVORK5CYII=\
+  texture: url("data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAoAAAAKCAYAAACNMs+9AAAAJklEQVQYlWP8z3DmPwMaYGQwYUQX\
+Y2IgEjANnEJGbILYPDgUPAMAz44F3Skt+skAAAAASUVORK5CYII=\
 ");
 }
diff --git a/testsuite/gsk/nodeparser/texture-scale-filters.ref.node b/testsuite/gsk/nodeparser/texture-scale-filters.ref.node
index 9570132..278f176 100644
--- a/testsuite/gsk/nodeparser/texture-scale-filters.ref.node
+++ b/testsuite/gsk/nodeparser/texture-scale-filters.ref.node
@@ -1,26 +1,26 @@
 texture-scale {
   bounds: 0 0 50 50;
-  texture: url("data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAoAAAAKCAYAAACNMs+9AAAAGElEQVQYlWP8z8Dwn4EIwESMolGF\
-1FMIAD2cAhL1w47oAAAAAElFTkSuQmCC\
+  texture: url("data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAoAAAAKCAYAAACNMs+9AAAAG0lEQVQYlWP8z8Dwn4EIwESMIhAY\
+VYgXEK0QAD2cAhLBNGStAAAAAElFTkSuQmCC\
 ");
 }
 texture-scale {
   bounds: 0 0 50 50;
-  texture: url("data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAoAAAAKCAYAAACNMs+9AAAAGElEQVQYlWP8z8Dwn4EIwESMolGF\
-1FMIAD2cAhL1w47oAAAAAElFTkSuQmCC\
+  texture: url("data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAoAAAAKCAYAAACNMs+9AAAAG0lEQVQYlWP8z8Dwn4EIwESMIhAY\
+VYgXEK0QAD2cAhLBNGStAAAAAElFTkSuQmCC\
 ");
 }
 texture-scale {
   bounds: 0 0 50 50;
   filter: nearest;
-  texture: url("data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAoAAAAKCAYAAACNMs+9AAAAGElEQVQYlWP8z8Dwn4EIwESMolGF\
-1FMIAD2cAhL1w47oAAAAAElFTkSuQmCC\
+  texture: url("data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAoAAAAKCAYAAACNMs+9AAAAG0lEQVQYlWP8z8Dwn4EIwESMIhAY\
+VYgXEK0QAD2cAhLBNGStAAAAAElFTkSuQmCC\
 ");
 }
 texture-scale {
   bounds: 0 0 50 50;
   filter: trilinear;
-  texture: url("data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAoAAAAKCAYAAACNMs+9AAAAGElEQVQYlWP8z8Dwn4EIwESMolGF\
-1FMIAD2cAhL1w47oAAAAAElFTkSuQmCC\
+  texture: url("data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAoAAAAKCAYAAACNMs+9AAAAG0lEQVQYlWP8z8Dwn4EIwESMIhAY\
+VYgXEK0QAD2cAhLBNGStAAAAAElFTkSuQmCC\
 ");
 }
diff --git a/testsuite/gsk/nodeparser/texture-scale-unknown-filter.ref.node b/testsuite/gsk/nodeparser/texture-scale-unknown-filter.ref.node
index 6a77d0f..ec4d363 100644
--- a/testsuite/gsk/nodeparser/texture-scale-unknown-filter.ref.node
+++ b/testsuite/gsk/nodeparser/texture-scale-unknown-filter.ref.node
@@ -1,6 +1,6 @@
 texture-scale {
   bounds: 0 0 50 50;
-  texture: url("data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAoAAAAKCAYAAACNMs+9AAAAKUlEQVQYlWP8z3DmPwMaYGQwYUQX\
-Y0IXwAUGUCGGoxkYGBiweXAoeAYAz44F3e3U1xUAAAAASUVORK5CYII=\
+  texture: url("data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAoAAAAKCAYAAACNMs+9AAAAJklEQVQYlWP8z3DmPwMaYGQwYUQX\
+Y2IgEjANnEJGbILYPDgUPAMAz44F3Skt+skAAAAASUVORK5CYII=\
 ");
 }
